sql/server/dev/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM skiplabs/skdb as skdb
+FROM skiplabs/skdb-base as skdb
```

Comment:
This should be skiplabs/skdb which builds on skiplabs/skdb-base. The dockerfile for this is in sql/Dockerfile. I refactored to make building for prod possible. I think I might have missed updating a script that needs to build this for the dev server?